lib/awful/hotkeys_popup/widget.lua

```diff
@@ -326,6 +370,7 @@ function widget.new(args)
         -- be presented to the user as-is. (This is useful for cheatsheets for
         -- external programs.)
         labels = args.labels or widget.labels,
+        modifier_order = args.modifier_order or widget.modifier_order,
```

Comment:
Seems like the naming is wrong:

```suggestion
        modifier_order = args.modifier_order or widget.modifier_sort_order,
```

Also, is it supposed to do something? `modifier_order` seems unused and `args.modifier_order` is not documented.

Reply:
I just forgot to remove it, it was meant for something but no longer is